iterators.tex

```diff
@@ -2991,41 +2986,39 @@
 
 \indexlibrary{\idxcode{insert_iterator}}%
 \begin{codeblock}
-namespace std { @\added{namespace experimental \{ namespace ranges \{ inline namespace v1 \{}@
+namespace std { namespace experimental { namespace ranges { inline namespace v1 {
   template <class Container>
-  class insert_iterator @\changed{:}{\{}@
-    @\removed{public iterator<output_iterator_tag, void, void, void, void> \{}@
-  @\changed{protected}{private}@:
-    Container* container;              @\added{// \expos}@
-    typename Container::iterator iter; @\added{// \expos}@
-
+  class insert_iterator {
   public:
-    @\changed{typedef Container}{using}@ container_type@\added{ = Container}@;
-    @\added{using difference_type = ptrdiff_t;}@
-    @\added{insert_iterator();}@
-    insert_iterator(Container& x, typename Container::iterator i);
-    insert_iterator@\removed{<Container>}@&
+    using container_type = Container;
+    using difference_type = ptrdiff_t;
+
+    insert_iterator();
+    insert_iterator(Container& x, iterator_t<Container> i);
```

Comment:
Actually, it was to brink the text body in line with a change that had already been made in the synopsis. I held off on the the other edits until LWG approved the change.